falco.yaml

```diff
@@ -695,6 +695,9 @@ webserver:
   # Can be an IPV4 or IPV6 address, defaults to IPV4
   listen_address: 0.0.0.0
   k8s_healthz_endpoint: /healthz
+  # Enable the metrics endpoint providing Prometheus values
+  # It will only have an effect if metrics.enabled is set to true as well.
+  prometheus_metrics_enabled: false
```

Comment:
@leogr do we need to enforce a feat status for this? Like `Incubating` ?

Reply:
It would be preferable, IMO. It is not a blocker for this PR anyway. We may reaudit all options later, but still before the 0.38 release. Does it make sense?

cc @falcosecurity/falco-maintainers

Reply:
@leogr proposing to merge it as is and address the follow up items in a new PR. I believe we need another touch up PR anyways.